### PR TITLE
[TG-2793] adding sweepers for acc tests

### PR DESCRIPTION
**Pull request by shmykyta**
> JIRA Ticket URL: https://twingate.atlassian.net/browse/TG-2793
> Notion spec URL:

## Changes
-  Add sweepers to acceptance test for automatically clean up the environment

### CODE REVIEW

File: twingate/resource_resource_sweeper_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		return fmt.Errorf("error getting client: %s", err)
+	}
+
+	resourceList, err := client.readAllResources()
+	if err != nil {
+		return fmt.Errorf("[INFO][SWEEPER_LOG] Nothing found in response: %s", err)
+	}
+
+	if len(resourceList) == 0 {
+		log.Printf("[INFO][SWEEPER_LOG] List %s is empty", resourceName)
```
Comment (qtheya) [inline twingate/resource_resource_sweeper_test.go:31]: you can return Errorf instead as in other return statements in this func

File: twingate/resource_remote_network_sweepers_test.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		return fmt.Errorf("error getting client: %s", err)
+	}
+
+	networkList, err := client.readAllRemoteNetwork()
+	if err != nil {
+		return fmt.Errorf("[INFO][SWEEPER_LOG] Nothing found in response: %s", err)
+	}
+
+	if len(networkList) == 0 {
+		log.Printf("[INFO][SWEEPER_LOG] List %s is empty", resourceName)
```
Comment (qtheya) [inline twingate/resource_remote_network_sweepers_test.go:31]: you can return Errorf instead as in other return statements in this func

Comment (shmykyta) [reply twingate/resource_remote_network_sweepers_test.go:32]: 19c9ec1

File: twingate/resource_resource_sweeper_test.go (hunk 1 above)
Comment (shmykyta) [reply twingate/resource_resource_sweeper_test.go:32]: 19c9ec1

**Comment by romankor** [comment]
@liorr please have a look at the PR

**Comment by romankor** [comment]
@liorr ping ...

File: twingate/client_connector.go [hunk 3]
```diff
@@ -51,6 +52,29 @@ func (client *Client) createConnector(remoteNetworkID string) (*Connector, error
 	return &connector, nil
 }
 
+func (client *Client) readTestConnectors() ([]string, error) { //nolint
```
Comment (liorr) [inline twingate/client_connector.go:55]: I think it is better just to have a `readConnectors` method and do the filtering on the caller `testSweepTwingateConnector` (same comment of other clients).

Comment (shmykyta) [reply twingate/client_connector.go]: Changed to common `readConnectors`: 6944e77ae8d0ef6510cc7fba00fd165b44b17d65

File: go.mod [hunk 4]
```diff
@@ -6,5 +6,6 @@ require (
 	github.com/Jeffail/gabs/v2 v2.6.1
 	github.com/hashicorp/go-retryablehttp v0.7.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.1
+	github.com/mattn/go-colorable v0.1.8 // indirect
```
[the remark this answers is not in the document]
Comment (shmykyta) [reply go.mod]: Removed after reinit go module: dc296559be07dd6322595da51e2de514bc7ef3fb

**Comment by shmykyta** [comment]
@liorr could you review please

**Review by liorr — APPROVED** [review]
GTM

**Pull request merged**